Configuration/Settings.yaml

```diff
@@ -34,6 +34,15 @@ Neos:
           'Neos.Neos.UI:HostOnlyStyles':
             resource: '${"resource://" + Neos.Ui.StaticResources.compiledResourcePackage() + "/Public/Styles/HostOnlyStyles.css"}'
             position: 'start 900'
+          'WorkSans:200':
```

Comment:
I don't know why we switched. As I first picked up font stuff it were already work-sans.
These is were I picked up:
https://github.com/neos/neos-ui/pull/761
https://github.com/neos/neos-ui/issues/690